client/app/expenses/components/UserDisplay.tsx

```diff
@@ -0,0 +1,15 @@
+import { MdAccountCircle } from 'react-icons/md'
+
+type UserDisplayProps = {
+  name: string
+  size: number
+}
+
+const UserDisplay = ({ name, size }: UserDisplayProps) => (
+  <div className='flex items-center gap-2'>
+    <MdAccountCircle size={size} />
+    <p className='md:text-lg'>{name}</p>
```

Comment:
You need to show dynamic colours based on the icon colour!
And `<h4>` is more correct for the name!

Reply:
Fixed here
https://github.com/HAK2024/fairy-share/commit/800ef0f1eb10ff8ccb9dd9b3b8cfaaa2e6593931